src/main/java/com/team4/testingsystem/dto/QuestionDTO.java

```diff
@@ -0,0 +1,86 @@
+package com.team4.testingsystem.dto;
+
+public class QuestionDTO {
+    private String questionBody;
+    private Boolean isAvailable;
+    private String creator;
+    private String level;
+    private String module;
+
+    public String getCreator() {
+        return creator;
+    }
+
+    public void setCreator(String creator) {
+        this.creator = creator;
+    }
+
+    public String getQuestionBody() {
+        return questionBody;
+    }
+
+    public void setQuestionBody(String questionBody) {
+        this.questionBody = questionBody;
+    }
+
+    public Boolean isAvailable() {
+        return isAvailable;
+    }
+
+    public void setAvailable(boolean available) {
+        isAvailable = available;
+    }
+
+    public String getLevel() {
+        return level;
+    }
+
+    public void setLevel(String level) {
+        this.level = level;
+    }
+
+    public String getModule() {
+        return module;
+    }
+
+    public void setModule(String module) {
+        this.module = module;
+    }
+
+    public static class Builder {
```

Comment:
Since this builder is used only to construct DTO from Question object, maybe constructor from Question is enough?

Reply:
in my opinion, using the builder is more convenient (and may be useful in the future)